pantheon-bundle/src/main/java/com/redhat/pantheon/extension/url/CustomerPortalUrlUuidProvider.java

```diff
@@ -30,10 +30,10 @@ public String generateUrlString(DocumentVariant variant) {
             ProductVersion pv = metadata.productVersion().getReference();
             StringBuilder sb = new StringBuilder(getHost(variant.getResourceResolver()));
             sb.append(URL_PREFIX)
-                    .append(getLocale(variant)).append("/")
-                    .append(getDocumentType(variant)).append("/")
+                    .append(getLocale(variant).replaceAll("_", "-").toLowerCase()).append("/") // turns en_US into en-us which is likely a customer portal requirement (need to confirm)
```

Comment:
we have a method in ServletUtils that transforms the locale to language tag. see
https://github.com/redhataccess/pantheon/blob/master/pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/module/VariantJsonServlet.java#L109